extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -1063,3 +1063,116 @@ async def get(self, objectId: str = None):
         except Exception:
             _err = traceback.format_exc()
             return self.error(f"failure: {_err}")
+
+
+class ZTFAlertsByCoordsHandler(ZTFAlertHandler):
+    @auth_or_token
+    def get(self):
+        """
+        ---
+        single:
+          description: Query Kowalski for alerts near specified coords
+          parameters:
+            - in: query
+              name: ra
+              required: true
+              schema:
+                type: float
+            - in: query
+              name: dec
+              required: true
+              schema:
+                type: float
+            - in: query
+              name: radius
+              required: true
+              schema:
+                type: float
```

Comment:
can we add descriptions of the parameters here. specifically, what are the units for radius (degrees or arcsec)?

Reply:
Done